plugins/rapid7_insightidr/plugin.spec.yaml

```diff
@@ -440,12 +528,14 @@ types:
       required: false
     status:
       title: Status
-      description: Holds a status code for the query, potentially different from the status code of the response
+      description: Holds a status code for the query, potentially different from the
+        status code of the response
```

Comment:
Ensure all the descriptions throughout are contained to a single line. I'm not sure if its a formatter causing this but lets keep them to one line regardless how long they are 😄